run.byclasses.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+
+<suite name="Positive/Nagative Tests" verbose="1">
+    <test name="Positive Tests">
+        <classes>
+            <class name = "hw_at_17.HomePageTests" />
+            <class name = "hw_at_17.SearchResultPageTests" >
+                <methods>
+                    <include name = "verifyPositiveSearchInput"/>
+                </methods>
+            </class>
+        </classes>
+    </test>
+    <test name="Negative Tests">
+        <classes>
+            <class name = "hw_at_17.SearchResultPageTests" />
+            <class name = "hw_at_17.SearchResultPageTests" >
+                <methods>
+                    <exclude name = "verifyPositiveSearchInput"/>
+                </methods>
+            </class>
+        </classes>
+    </test>
+</suite>
```

Comment:
You need to create 3 separate .xml files
![image](https://github.com/user-attachments/assets/f2779cb0-d234-40c4-8e52-2a4b86030710)

Reply:
fixed